python/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.10)
 
+set (CMAKE_BUILD_TYPE Debug)
```

Comment:
Please no. Debug must be only when requested.

Reply:
I think this just got left in by mistake!